analysed-packages/atlassian/version-3.41.3/README.md

```diff
@@ -0,0 +1,20 @@
+## Download Location
+
+https://github.com/atlassian-api/atlassian-python-api
+
+## Package URL (purl)
+
+pkg:github/atlassian-api/atlassian-python-api@3.41.3
+
+## Creator
+
+Kanzlei Jun on behalf of AUDI AG
+
+## Reviewers
+
+The information was reviewed by:
+
+
+## Comment
+
+Only the Python packages of the components have been curated, not the entire packages found in the download location.
```

Comment:
In case the same applies here as for altgraph, this is an inconsistency, which makes the analysis not usable because you have probably installed the package via "pip install" fetching the package from pypi but do you have a proof that this is exactly the same as the package specified in the download link?
To be honest and I am sorry but I cant accept such a potential source of inconsistent data.
I also did a 
`pipinstall atlassian-python-api==3.41.3`
 in order to verify that what is installed via pip is not the package available on GitHub, in the best case it is a subset and in the worst case it contains different files or different file content (i did not verify the file hashes).